### ISSUE TITLE
experience browser/mobile 

### ISSUE THREAD

**Issue by q-ki** [nextcloud/spreed#6969]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
Hi there. 

Searched the past issues ... cannot find anything similar ... so here it is ... 

**** invitation to a talk as a nc-non-registered user via browser on smartphone**** 
 
1) as in my previous issue described ... you have to turn your phone into landscape-mode because the layout in the portrait-mode is screwed ...if you close the details-tab it is fine ... but before that everything looks not good 
2) there is no sound/ringing when the other side starts the conversation ... only a visual clue that the conversation has started ...there is only sound/ringing if you start the call/conversation 
3) if you are in landscape-orientation and fullscreen you always see the big red button (in german "Anruf verlassen" ... so maybe "leave/end call" in english) ... this is dominating the whole screen 

**Describe the solution you'd like**
1) optimize css for portrait-orientation and/or deactivate details-tab on default when first entering
2) in addition to the visual clue a ringing/sound when the other side starts the call/conversation 
3) all buttons fade out after a few seconds and come back when you touch the screen 

Cheers ...
qki

**Comment by nickvergessen** [comment]
Duplicate of your own issue from yesterday? https://github.com/nextcloud/spreed/issues/6957

**Comment by q-ki** [comment]
Yes... I didn´t know what to do with the other post. This one evolved out of it by more testing on my side.
 So your action to close the other one is right. Thanks.
